CONTRIBUTING_ru-RU.md

```diff
@@ -1,63 +1,64 @@
-- Добавьте новые функции в ветвь master и исправьте ошибку в последней активной ветви (master, beta и stable), в которой есть эта ошибка.  
-- Пожалуйста, напишите [сообщения к правке](https://chris.beams.io/posts/git-commit/). Также рекомендуется написать [обычные правки](https://www.conventionalcommits.org/ru). Вы можете использовать [commitizen/cz-cli](https://github.com/commitizen/cz-cli) чтобы написать сообщения к правке. Однако правила необязательны, главное, чтобы и другие, и вы сами в будущем понимали, что было изменено и почему эти изменения сделаны.
-- Используйте Clang Format 9 (введите `clang-format --version` чтобы проверить свою версию, Вам может потребоваться указать путь к исполняемому файлу clang-format для вашей IDE (например, версия clang-format по умолчанию в коде VS может быть clang-format-6) и к файлу `.clang-format` в корневом каталоге, чтобы форматировать код.
-- Не удаляйте функции, если в этом нет необходимости. Вместо этого добавьте опцию отключения какой-либо функции.
-- Все время поддерживайте обратную совместимость файла настроек.
-- Стоит поддерживать программу под Windows, Linux и Mac OS.
-- Следует поддерживать как светлые, так и темные системные темы.
- - Все должны подчиняться правилам [Code of Conduct](CODE_OF_CONDUCT.md) для того чтобы построить сообщество, свободное от притеснений.
-
-## FAQ
-
-- Могу ли я сделать все корректно, если я новичок?
-  - Все когда-то были новичками. Не волнуйтесь, наши крутые участники помогут вам, когда вы столкнетесь с трудностями. Если Вы :heart: open source и будете стараться, у Вас все получиться.
-- Нужно ли мне знать CMake?
-  - Нет, нужно только знать, как делается [сборка из исходного кода](doc/INSTALL_ru-RU.md#Сборка-из-исходного-кода).
-- Как добавить настройку?
-  - Вы должны добавить пользовательский интерфейс в `ui/preferencewindow.ui`, добавить set/get функции в `include/Settings/SettingsManager.hpp` и `src/Settings/SettingsManager.cpp`, добавить код в `applySettingsToui`, `extractSettingsFromui` и `isUnsavedChanges` в `src/preferencewindow.cpp`. Копипастинг очень полезен.
-- Некоторые файлы отсутствуют при компиляции.
-  - Если тесты CI пройдены, то из-за того, что обновляются подмодули. Вы можете выполнить следующие команды:
-
-    ```sh
-    git add .
-    git reset --hard
-    git submodule update --init
-    ```
-
-    Вы также можете попробовать `git rm` все подмодули, а затем установить их снова. Можно даже удалить репозиторий на вашем диске и клонировать его снова.
+# Внимание! Данный перевод может быть устаревшим и не соответствовать полноценному переводу актуальной версии документа на английском языке.

```

Comment:
I think the warning should be a normal line instead of a header and contains a link to the English version.

Same for other documents.